modules/network/src/test/java/org/apache/ignite/internal/network/DefaultMessagingServiceTest.java

```diff
@@ -525,6 +539,8 @@ private Services createMessagingService(
 
         messagingService.setConnectionManager(connectionManager);
 
+        messagingService.start();
```

Comment:
Why did this line not exist before?

Reply:
Probably because we didn't test for timeouts